src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -7,9 +7,12 @@ import TextWidget from './TextWidget';
  * instead
  */
 export default class PhoneNumberWidget extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { val: props.value };
+  state = { val: this.props.value, firstUpdate: true };
+
+  componentDidUpdate(prevProps) {
+    if (this.state.firstUpdate && this.props.value !== prevProps.value) {
+      this.handleChange(this.props.value);
```

Comment:
It seems like this will only call `handleChange` once, and then `handleChange` will update the state to make sure it isn't called again when the component updates.  Is there a reason that we don't want it called each time?

Reply:
What happens in that case is the value in the text field can change unexpectedly as the user is entering their phone number; ie the value that's stripped of anything but numbers will get set as the value of the text field. 

![change](https://user-images.githubusercontent.com/20728956/75828339-73d7fd80-5d60-11ea-9734-1f946e3fefb8.gif)